### DISCUSSION TITLE
How can I upgrade over command line?

### DISCUSSION THREAD

**Discussion by techrivertree** [mealie-recipes/mealie#3924] [Q&A]
I am able to upgrade the RPi software with sudo apt-y update and sudo apt -y upgrade but that is not updating Mealie. I read through the install instructions and FAQ. How can I upgrade from v 1.6 to 1.10 using the command line?

**Comment by Kuchenpirat** [comment]
i assume you are running mealie through a docker container. docker uses another system to update programms. 

- check in your `docker-compose.yml` file if there is a version specified after the image tag. (E.g: _image: ghcr.io/mealie-recipes/mealie:**v1.6.0**_), or whether it has a self updating image tag (_latest_ or _nightly_). If it is a dedicated version change the tag to the version you want to download. If not you can skip this step. 
- Open a command prompt in the folder your docker-compose.yml file is stored. 
- execute `docker compose pull` to download the new image
- execute `docker compose up -d` to update the container. 

After that you might want to look into `docker image prune` or `docker system prune` to clean up old images / files

**Comment by techrivertree** [comment]
Mealie is installed on an RPi. Using docker-compose.yml does not work, 

PiMeal@PiMeal:~ $ docker-compose.yml
-bash: docker-compose.yml: command not found

**Reply by Aechay** [reply]
If you want to open the docker-compose.yml file, try running `nano docker-compose.yml` or `sudo nano docker-compose.yml` from the command line inside the folder that contains your mealie instance. From there like @Kuchenpirat suggested, you can run `docker compose down && docker compose pull && docker compose up -d` to stop, update, and restart you container.
